docs/Definition_of_ Ready.md

```diff
@@ -56,4 +57,10 @@ so that [because - why this is important? What problem is this solving?]
 ## Not included:
 -
 -
+
+## Technical Analysis
+- [ ] 
+- [ ] 
+- [ ] 
```

Comment:
Do we want this empty section in our DOR?

Reply:
Yes, this will be part of the template.